src/components/Whiteboard/WhiteboardHost.tsx

```diff
@@ -113,8 +113,9 @@ export const WhiteboardHostConnected = () => {
   const { state: presentationState } = usePresentationMode();
   const isPresenting = presentationState.type === 'presenting';
   const isViewingPresentation = presentationState.type === 'presentation';
-  const isViewingPresentationInEditMode =
-    isViewingPresentation && presentationState.isEditMode;
+
```

Comment:
this empty line not needed, a bit nitpicky but would be consistent across the code